x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/constants.ts

```diff
@@ -163,3 +166,29 @@ export const CUSTOM_CONNECTORS = CONNECTORS.filter(({ isNative }) => !isNative);
 export const NATIVE_CONNECTORS = CONNECTORS.filter(({ isNative }) => isNative);
 
 export const BETA_CONNECTORS = CONNECTORS.filter(({ isBeta }) => isBeta);
+
+export const getConnectorTemplate = ({
+  apiKeyData,
+  connectorData,
+  host,
+}: {
+  apiKeyData: ApiKey | undefined;
+  connectorData: {
+    id: string;
+    service_type: string | null;
+  };
+  host?: string;
+}) => dedent`connectors:
+  -
+    connector_id: "${connectorData.id}"
+    service_type: "${connectorData.service_type || 'changeme'}"${
+  apiKeyData?.encoded
+    ? `
+    api_key: "${apiKeyData?.encoded}"`
+    : ''
+}
+
+  elasticsearch:
+    host: "${host || 'https://locahost:9200'}"
```

Comment:
@leemthompo it will use cloud when available.

Reply:
Ah great, thanks for confirming @efegurkan ❤️